packages/vite/src/client/overlay.ts

```diff
@@ -213,7 +214,7 @@ export class ErrorOverlay extends HTMLElement {
     super()
     this.root = this.attachShadow({ mode: 'open' })
 
-    this.root.appendChild(template)
+    this.root.appendChild(createTemplate())
```

Comment:
I was thinking we could keep a `let template`, then do
```suggestion
    this.root.appendChild(template ??= createTemplate())
```
so we still reuse the template, but we do it lazily

Reply:
Updated 👍